Formula/fn.rb

```diff
@@ -0,0 +1,52 @@
+class Fn < Formula
+  desc "Command-line tool for the fn project"
+  homepage "https://fnproject.github.io"
+  url "https://github.com/fnproject/cli/archive/0.3.84.tar.gz"
+  sha256 "495c93a955b2a4cfb0a2ea4a32cd150fc2a98192051d99a51a1f63bddfe178fc"
+
+  depends_on "go" => :build
+  depends_on "glide" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GLIDE_HOME"] = HOMEBREW_CACHE/"glide_home/#{name}"
+    dir = buildpath/"src/github.com/fnproject/cli"
+    dir.install Dir["*"]
+    cd dir do
+      system "glide", "install", "-v", "--force", "--skip-test"
+      system "go", "build", "-o", "#{bin}/fn"
+      prefix.install_metafiles
+    end
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/fn --version")
```

Comment:
This can probably be dropped now you've added the stronger test below.

Reply:
I think this is still important to ensure we didn't mess up a release, so left it in. Can't do any harm.